languages/elixir/exercises/concept/charlists/.docs/instructions.md

```diff
@@ -0,0 +1,41 @@
+You are working as a system administrator for a big company in Münich, Germany. One of your responsibilities is managing email accounts.
+
+You have been hearing complaints from people saying they are unable to write emails to Mr. Müller. You quickly realize that most of the company uses an old email client that doesn't recognize `müller@firma.de` as a valid email address because of the non-latin character.
+
+Telling people to give up their favorite old email client is a lost battle, so you decide to create sanitized aliases for all email accounts.
```

Comment:
Love this theme! My previous employer had a German branch so I can empathize with this story :)